x-pack/plugins/lists/server/services/exception_lists/utils/import/sort_import_by_namespace.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ImportExceptionListItemSchemaDecoded,
+  ImportExceptionListSchemaDecoded,
+} from '@kbn/securitysolution-io-ts-list-types';
+
+/**
+ * Helper to sort exception lists by namespace type
+ * @param exceptions {array} exception lists to sort
+ * @returns {array} tuple of agnostic and non agnostic lists
+ */
+export const sortListsImportsByNamespace = (
+  exceptions: ImportExceptionListSchemaDecoded[]
+): [ImportExceptionListSchemaDecoded[], ImportExceptionListSchemaDecoded[]] => {
```

Comment:
I'd add return types if possible to these functions

Reply:
It's there as `[ImportExceptionListSchemaDecoded[], ImportExceptionListSchemaDecoded[]]`.